app/models/book.rb

```diff
@@ -1,3 +1,8 @@
 class Book < ApplicationRecord
-  
+  validates_presence_of :title
+  has_many :reviews
+
+  def average_rating
+    reviews.average_rating
+  end
```

Comment:
This works, but `reviews.average(:rating)` should have worked as well.

Reply:
I like that you still wrapped it in a book method rather than chaining from from `book.reviews.average_rating`. Good use of Law of Demeter and keeping logic out of your views.